acl/src/main/java/org/springframework/security/acls/jdbc/JdbcAclService.java

```diff
@@ -88,7 +88,7 @@ public JdbcAclService(DataSource dataSource, LookupStrategy lookupStrategy) {
 	// ========================================================================================================
 
 	public List<ObjectIdentity> findChildren(ObjectIdentity parentIdentity) {
-		Object[] args = { parentIdentity.getIdentifier(), parentIdentity.getType() };
+		Object[] args = { "" + parentIdentity.getIdentifier(), parentIdentity.getType() };
```

Comment:
What about calling `toString()` instead of doing string concatination? Or may the objects be `null`?

```suggestion
		Object[] args = { parentIdentity.getIdentifier().toString(), parentIdentity.getType() };
```

Reply:
@mangei Thanks! 
No the identifier could never be null, I've adapted the code accordingly!